cucumber-messages/specs/receiving/TestRunStarted.feature

```diff
@@ -0,0 +1,17 @@
+Feature: Receiving TestRunStarted messages
+
+Scenario: Handling a message queue with no TestRunStarted message
+  Given there is no TestRunStarted message in the message queue
+  When the messsage queue is processed
+  Then there is an error
+  And the queue processing is stopped
+
+Scenario: Handling a message queue with a singe TestRunStarted message
+  Given there is a TestRunStarted message in the message queue
+  When the message queue is processed
+  Then the start of the test run was recognized
+
+Scenario: Handling a message queue with two TestRunStarted messages
+  Given there are two TestRunStarted messages in the message queue
+  When the message queue is processed
+  Then the second TestRunStarted message has been ignored
```

Comment:
Should this check that the start of the run was recognized?

Reply:
Yes, this check should not be omitted. @SabotageAndi what do you think?

Reply:
I've added the same Then statement as in the scenario above, which should clarify the behavior.